From 133af70d579d8800db95551ffe84502f46442277 Mon Sep 17 00:00:00 2001
From: Robin Lee <robinlee.sysu@gmail.com>
Date: Thu, 12 Jan 2023 18:08:23 +0800
Subject: [PATCH 03/14] pcre2: version 10.42 does not support jit for
 loongarch64

---
 elements/bootstrap/pcre2.bst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elements/bootstrap/pcre2.bst b/elements/bootstrap/pcre2.bst
index fe793a918..89b9fd7c0 100644
--- a/elements/bootstrap/pcre2.bst
+++ b/elements/bootstrap/pcre2.bst
@@ -24,7 +24,7 @@ variables:
   (?):
   - (target_arch == "x86_64" or target_arch == "i686"):
       local_flags: -mshstk
-  - target_arch != "riscv64":
+  - (target_arch != "riscv64" and target_arch != "loongarch64"):
       arch_options: "--enable-jit"
 
   conf-local: >-
-- 
2.41.0

