From 0eb55bcf2daa488d76714f9478ba1e4a38ae5ff0 Mon Sep 17 00:00:00 2001
From: Robin Lee <robinlee.sysu@gmail.com>
Date: Wed, 11 Oct 2023 23:10:19 +0800
Subject: [PATCH 04/13] rsync: update gnu configs for loongarch64

---
 elements/components/rsync.bst | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/elements/components/rsync.bst b/elements/components/rsync.bst
index 321fe5c90..747d76584 100644
--- a/elements/components/rsync.bst
+++ b/elements/components/rsync.bst
@@ -16,5 +16,13 @@ variables:
     --disable-simd
     --disable-md2man
 
+config:
+  (?):
+  - target_arch == "loongarch64":
+      configure-commands:
+        (<):
+        - |
+          cp -v %{datadir}/gnu-config/config* .
+
 (@):
 - elements/include/rsync-source.yml
-- 
2.41.0

