From bf15b5b7fbd21853c2a20811c5620669cb8c5285 Mon Sep 17 00:00:00 2001
From: Robin Lee <robinlee.sysu@gmail.com>
Date: Wed, 11 Oct 2023 23:10:31 +0800
Subject: [PATCH 05/14] sqlite: update gnu configs for loongarch64

---
 elements/components/sqlite.bst | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/elements/components/sqlite.bst b/elements/components/sqlite.bst
index 2be8ba76c..00f2d971c 100644
--- a/elements/components/sqlite.bst
+++ b/elements/components/sqlite.bst
@@ -22,6 +22,13 @@ config:
     - sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' %{build-dir}/libtool
     - sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' %{build-dir}/libtool
 
+  (?):
+  - target_arch == "loongarch64":
+      configure-commands:
+        (<):
+        - |
+          cp -v %{datadir}/gnu-config/config* .
+
 public:
   bst:
     split-rules:
-- 
2.41.0

