From c8fcb76484148bebc0f14e41b188d1e687e75eba Mon Sep 17 00:00:00 2001
From: Robin Lee <robinlee.sysu@gmail.com>
Date: Wed, 11 Oct 2023 23:10:31 +0800
Subject: [PATCH 06/13] sqlite: update gnu configs for loongarch64

---
 elements/components/sqlite.bst | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/elements/components/sqlite.bst b/elements/components/sqlite.bst
index 2be8ba76c..36e06cec1 100644
--- a/elements/components/sqlite.bst
+++ b/elements/components/sqlite.bst
@@ -18,6 +18,12 @@ variables:
 
 config:
   configure-commands:
+    (?):
+    - target_arch == "loongarch64":
+        (<):
+        - |
+          cp -v %{datadir}/gnu-config/config* .
+
     (>):
     - sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' %{build-dir}/libtool
     - sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' %{build-dir}/libtool
-- 
2.41.0

