From a157c62120ee873a8a0a18e3d2ea4fbde7cb3afa Mon Sep 17 00:00:00 2001
From: Robin Lee <robinlee.sysu@gmail.com>
Date: Thu, 12 Jan 2023 20:45:15 +0800
Subject: [PATCH 08/13] libseccomp: use devel source for loongarch64

---
 elements/components/libseccomp.bst | 24 +++++++++++++++++++-----
 1 file changed, 19 insertions(+), 5 deletions(-)

diff --git a/elements/components/libseccomp.bst b/elements/components/libseccomp.bst
index af048c74e..d87a29b47 100644
--- a/elements/components/libseccomp.bst
+++ b/elements/components/libseccomp.bst
@@ -15,8 +15,22 @@ public:
         (>):
         - '%{libdir}/libseccomp.so'
 
-sources:
-- kind: git_repo
-  url: github:seccomp/libseccomp.git
-  track: v*
-  ref: v2.5.4-0-gf33f95014b36f97b42f0c2290e96d5c31647ed10
+(?):
+- target_arch != "loongarch64":
+    sources:
+    - kind: git_repo
+      url: github:seccomp/libseccomp.git
+      track: v*
+      ref: v2.5.4-0-gf33f95014b36f97b42f0c2290e96d5c31647ed10
+- target_arch == "loongarch64":
+    config:
+      install-commands:
+        (>):
+        - |
+          sed -i 's/Version:.*/Version: 2.6.0/' %{install-root}%{libdir}/pkgconfig/libseccomp.pc
+
+    sources:
+    - kind: git_repo
+      url: github:seccomp/libseccomp.git
+      track: main
+      ref: v2.2.0-496-gf1c3196d9b95de22dde8f23c5befcbeabef5711c
-- 
2.41.0

